src/stream.ts

```diff
@@ -30,6 +37,14 @@ export class Stream extends EventEmitter {
     readonly sampleRate: number;
     readonly channelCount: number;
     private almostFinishedTrigger: number;
+    private byteLength: number;
+    private cacheSize: number = 0;
+    private playedBytes = 0;
+    private chunk: Buffer;
+    private _readablePaused = false;
+    remoteTime?: RemotePlayingTimeCallback;
+    remoteLagging?: RemoteLaggingCallback;
+    overflowCallback?: (pause: boolean) => void;
```

Comment:
What's this for?

Reply:
some of codes was from @Laky-64 project
but in my implementation i didn't use that variables

original source: https://github.com/pytgcalls/pytgcalls/blob/master/src/stream.ts